tests/base/src/test/java/org/wildfly/security/sasl/oauth2/OAuth2SaslClientV11Test.java

```diff
@@ -497,10 +497,7 @@ public void testWithResourceOwnerCredentialsInCredentialStoreUsingConfiguration(
         assertTrue(saslClient.isComplete());
     }
 
-    @Test
-    public void failedResourceOwnerCredentialsUsingConfiguration() throws Exception {
-        SaslClient saslClient = createSaslClientFromConfiguration(URI.create("protocol://test9.org"));
-
+    public void failedCredentialsUsingConfiguration(SaslClient saslClient) throws Exception {
```

Comment:
@ericrleung Just a minor, you can pass "String" serverURI to this method and create a SaslClient there

Reply:
@Skyllarr What should I change the name to?

Reply:
Maybe testInvalidCredentials or testInvalidCredentialsUsingConfiguration?

Reply:
yes those sound good